app/models/ingest_job.rb

```diff
@@ -418,6 +419,12 @@ def generate_success_email_array
           end
         end
         message << "Total points in cluster: #{cluster.points}"
+        # notify user that subsampling is about to run and inform them they can't delete cluster/metadata files
+        if cluster.can_subsample? && self.study.metadata_file.present?
+          message << "This cluster file will now be processed to compute representative subsamples for visualization."
+          message << "You will receive an additional email once this has completed."
+          message << "While subsamples are being computed, you will not be able to remove this cluster file or your metadata file."
+        end
```

Comment:
Possible other language below -- feel free to pick and choose sentences/phrases you like.   

This cluster file is now being subsampled for visualization. Please do not delete your cluster or metadata file while this process is occurring

You will receive an email once visualization processing is complete, letting you know this cluster is ready for viewing.

Reply:
My only issue with "please do not delete your file" is that it makes it seem as though they have the option, when in fact they do not.  Also, with the second option - they can view the cluster before it is subsampled, just only at full resolution.  Therefore I feel like both options are misleading.

Since I need to make further updates anyway, I'll ruminate over the weekend and see if I come up with something better.